src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int randomIndex = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[randomIndex];
+        return color.toString();
```

Comment:
You should use the 'name()' method to get the String representation of the enum constants instead of 'toString()', to avoid issues if 'toString()' is overridden.